src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -434,10 +425,17 @@ function ComposerWithSuggestions({
     }, [focus, prevIsFocused, prevIsModalVisible, isFocused, modal.isVisible, isNextModalWillOpenRef]);
 
     useEffect(() => {
+        // Indicate that draft has been created.
+        if (previousCommentRef.current.length === 0 && value.length !== 0) {
```

Comment:
Since we call this function anyway, could we do next:
```javascript
const hasDraft = previousCommentRef.current.length === 0 && value.length !== 0

Report.setReportWithDraft(reportID, hasDraft);
```

Reply:
isn't the same logic, because seems that the next if, check the `value` so the first can be false because of `previousCommentRef.current.length === 0` and you will be calling `setReportWithDraft` even when `value` isn't 0